airflow/providers/amazon/aws/hooks/logs.py

```diff
@@ -99,7 +97,7 @@ def get_log_events(
 
             yield from events
 
-            if 'nextForwardToken' in response:
+            if next_token != response['nextForwardToken']:
```

Comment:
Should we use `response.get('nextForwardToken', None)` to avoid any chance of a  KeyError here, or do we know for sure that will always be included in the response?

Reply:
Probably OK to just fail if the response format changes. An exception raised here (assuming the hook is used in a DAG) would eventually be caught by the test runner and logged to the task logs, which would clearly signal the problem (the API has an unexpected change in format). If we silently fall back to None, it would be more difficult to debug later if the API changed format unexpected (missing `nextForwardToken`), or actually returned `"nextForwardToken": null` (which is expected and signals a valid scenario, indicating we or the user may have logical bugs in the workflow).

Reply:
@ferruzzi @uranusjr Is there any change required in this PR or is it good to go?

Reply:
uranusjr covered my concern, but I'm also not a committer so my approval doesn't carry much (any) weight.